openeo_udp/eurac_pv_farm_detection/udf_eurac_pvfarm_onnx.py

```diff
@@ -0,0 +1,196 @@
+import functools
+import os
+import sys
+import zipfile
+from typing import Dict
+
+import numpy as np
+import requests
+import xarray as xr
+
+# TODO move standard code to UDF repo
+
+# Fixed directories for dependencies and model files
+DEPENDENCIES_DIR = "onnx_dependencies"
+MODEL_DIR = "model_files"
+
+
+def download_file(url, path):
+    """
+    Downloads a file from the given URL to the specified path.
+    """
+    response = requests.get(url, stream=True)
+    with open(path, "wb") as file:
+        file.write(response.content)
+    print(f"Downloaded file from {url} to {path}")
```

Comment:
UDFs should not use print. I don't think the user is able to see that anyway. And I even think this has potential to break the batch job logging.

In any case, we should not promote using "print" from UDFs